### DISCUSSION TITLE
ValueError with change env WEBLATE_API_RATELIMIT_USER in Docker

### DISCUSSION THREAD

**Discussion by a1iante** [WeblateOrg/weblate#8674] [Q&A] [answered]
Hi,

I installed Weblate in Docker following the guide: https://docs.weblate.org/en/latest/admin/install/docker.html

When changing to another values for the WEBLATE_API_RATELIMIT_USER env, we get a ValueError:

![image](https://user-images.githubusercontent.com/3246330/214590304-993150b0-1804-49a4-8986-d83f53912ab2.png)

Anyone know what could be the cause of this issue?
Thanks!

**Comment by nijel** [comment, marked as answer]
It should contain a string like `5000/hour`, see https://docs.weblate.org/en/latest/admin/install/docker.html#envvar-WEBLATE_API_RATELIMIT_USER. If you are specifying that in YAML, you might need to quote it.

**Reply by a1iante** [reply]
My bad, I have used to API limits using integer values. Great work, thanks for an answer!

**Reply by nijel** [reply]
I've added validation in 09c74eb7ec so that it fails earlier.
